productize.sh

```diff
@@ -17,5 +17,7 @@ END
 sed -i \
     -e 's/registry.svc.ci.openshift.org\/openshift\/release:golang-1.13/openshift\/golang-builder:1.13/' \
     -e 's/FROM registry.access.redhat.com\/ubi8\/ubi-minimal:[0-9.]*/FROM ubi8-minimal:8-released/' \
+    -e 's/COPY . /COPY keycloak-operator-*.tar.gz /' \
+    -e 's,RUN cd /src ,RUN cd /src \&\& tar -x --strip-components=1 -f keycloak-operator-*.tar.gz ,' \
```

Comment:
Wow, I've never seen `sed` syntax with `,`. Just double checking if this is what you need here (and is not a typo)?

Reply:
Technically you can use any character you want (certainly any symbol character anyway) after the s, and that becomes the delimiter character. In this case I'm dealing with paths in the pattern, so I need something other than `/` as the delimiter to avoid the pain of escaping those.

For example:

```bash
[aszczucz@tyrfing ~]$ sed -e 's,/home/,/etc/,' <<<"/home/alex/"
/etc/alex/
```

Reply:
Cool! I didn't know that! Thanks for clarifying!